cmd/lotus-chainwatch/scheduler/scheduler.go

```diff
@@ -44,7 +44,7 @@ func (s *Scheduler) Start(ctx context.Context) {
 		if err := refreshTopMinerByBaseReward(ctx, s.db); err != nil {
 			log.Errorf(err.Error())
 		}
-		refreshTopMinerCh := time.NewTicker(6 * time.Hour)
+		refreshTopMinerCh := time.NewTicker(30 * time.Second)
```

Comment:
6 hours is not frequent enough per conversation with @momack2

Reply:
LGTM, though I'm not sure about the impact of increased load.